src/solvers/smt2_incremental/smt2_incremental_decision_procedure.cpp

```diff
@@ -255,6 +255,7 @@ smt2_incremental_decision_proceduret::smt2_incremental_decision_proceduret(
     smt_set_option_commandt{smt_option_produce_modelst{true}});
   solver_process->send(smt_set_logic_commandt{smt_logic_allt{}});
   solver_process->send(object_size_function.declaration);
+  solver_process->send(is_dynamic_object_function.declaration);
```

Comment:
⛏️ Personal opinion: I know `smt_is_dynamic_objectt` is default-constructed, but readability would benefit if we explicitly construct it 5 lines above as in `object_map{initial_smt_object_map()}`.